privilege/privileges/privileges_test.go

```diff
@@ -2464,3 +2464,11 @@ func TestPlacementPolicyStmt(t *testing.T) {
 	mustExec(t, se, dropStmt)
 
 }
+
+func TestIssue27710(t *testing.T) {
```

Comment:
I know there are some historical exceptions to it - but we recommend naming the test as something useful, and including the issue as a comment:
```suggestion
// Issue 27710
func TestCaseSensitivity(t *testing.T) {
```

Reply:
thanks~  it fixed!